src/classes/contexts/CommandContext.ts

```diff
@@ -58,26 +58,43 @@ export class CommandContext implements CommandContextBuilder {
 		return this.message.author;
 	}
 
-	public async react(emoji: EmojiIdentifierResolvable) {
-		return await this.message.react(emoji);
-	}
-
-	public async removeReactions() {
-		return this.message.reactions.removeAll();
+	public async react(emoji: EmojiIdentifierResolvable|EmojiIdentifierResolvable[]) {
+		if (emoji instanceof Array) {
+			for (const e of emoji) {
+				await this.message.react(e);
+			}
+		} else return await this.message.react(emoji);
```

Comment:
`return await` is useless for an async method.

Reply:
removed